packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -1425,6 +1432,18 @@ class _SelectableRegionContainerDelegate extends MultiSelectableSelectionContain
 /// This class optimize the selection update by keeping track of the
 /// [Selectable]s that currently contain the selection edges.
 abstract class MultiSelectableSelectionContainerDelegate extends SelectionContainerDelegate with ChangeNotifier {
+  /// Creates a [MultiSelectableSelectionContainerDelegate].
+  ///
+  /// The [selectionSeparator] is used to separate the text of each selectable in
+  /// [selectables].
+  MultiSelectableSelectionContainerDelegate({
+     this.selectionSeparator = '',
+  });
+
+  /// The separator used to separate the text of each selectable in
+  /// [selectables].
+  final String selectionSeparator;
```

Comment:
probably use a getter instead of final property will make it easier to apply to _ScrollableSelectionContainerDelegate

Reply:
also probably name this to selectedContentSepatator

Reply:
I need clarification on the other points before I can do anything with this